README.md

```diff
@@ -4,20 +4,37 @@ The code to replace the processing logic of Bichard 7.
 
 ## Contents
 
-- [Quickstart](#quickstart)
-  - [Pre-Requisites](#pre-requisites)
-  - [Booting the infrastructure](#booting-the-infrastructure)
-  - [Running legacy Bichard in debug mode](#running-legacy-bichard-in-debug-mode)
-  - [Building on an M1 Mac](#building-on-an-m1-mac)
-- [Publishing package updates](#publishing-package-updates)
-- [Testing](#testing)
-- [Excluding Triggers](#excluding-triggers)
-- [Comparing New and Old Bichard](#comparing-new-and-old-bichard)
-  - [Checking a comparison file](#checking-a-comparison-file)
-  - [Checking a comparison file on old Bichard](#checking-a-comparison-file-on-old-bichard)
-  - [Comparing outputs locally](#comparing-outputs-locally)
-  - [Configuration](#configuration)
-- [Conductor](#conductor)
+- [Bichard 7 Core](#bichard-7-core)
+  - [Contents](#contents)
```

Comment:
I don't think these links are useful and can be removed.

Reply:
I like having a Table of Contents 🤷‍♂️

Reply:
Not saying I don't like a table of contents, just those specific two links. (I was the one who added a TOC in the first place! 😝)